test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -503,6 +503,50 @@ Describe "Get-Help should accept arrays as the -Parameter parameter value" -Tags
     }
 }
 
+Describe "Get-Help for function parameter should be consistent" -Tags 'CI' {
+    BeforeAll {
+        $test1 = @'
+            function test1 {
+                param (
+                    $First
+                )
+            }
+'@
+        $test2 = @'
+            function test2 {
+                param (
+                    $First,
+                    $Second
+                )
+            }
+'@
+        $test1Path = Join-Path $TestDrive "test1.ps1"
+        New-Item -ItemType File -Path $test1Path -Force > $null
+        Set-Content -Path $test1Path -Value $test1
+
+        $test2Path = Join-Path $TestDrive "test2.ps1"
+        New-Item -ItemType File -Path $test2Path -Force > $null
+        Set-Content -Path $test2Path -Value $test2
+
+        Import-Module $test1Path
+        Import-Module $test2Path
+    }
+
+    AfterAll {
+        Remove-Module -Name "test1"
+        Remove-Module -Name "test2"
+
+        Remove-Item $test1Path -Force -ErrorAction SilentlyContinue
```

Comment:
Don't need the `Remove-Item`. `$TestDrive` is destroyed automatically

Reply:
Done :)